framework/src/st_things/things_stack/things_stack.c

```diff
@@ -640,7 +638,7 @@ int things_return_user_opinion_for_reset(int b_reset_start)	// b_reset_start : U
 
 	int result = 0;
 
-	if (b_reset_start == true && b_reset_continue_flag == true) {	// User allow Reset-Starting.
+	if (b_reset_start == 1 && b_reset_continue_flag == true) {	// User allow Reset-Starting.
```

Comment:
i think you need to change the variable name.
Variable names beginning with b_ are of type bool.

Reply:
Okay. 
Note: It's existing code and not reported by static analysis fix.